honeybee_standards/_util/_program_type.py

```diff
@@ -0,0 +1,177 @@
+# coding=utf-8
+"""Clean the program type data."""
+import os
+import json
+
+
+def clean_space_types(source_filename, dest_directory, vintage):
+    """Process an OpenStudio Standards Space Type dictionary and write out a clean version.
+
+    Specifically, this method performs 3 cleaning operations:
+    * Discounting odd space types that are not generally aplicable but are intsead
+        specific to the DoE reference buildings. (eg. 'Apartment_topfloor_NS')
+    * Renaming space types that use naming conventions that are not consistent with the
```

Comment:
I agree. I am planning to have Matt review the code as part of the few hours he has for IN2. This will let him know the types of clean-up that we are doing on the data and it will give him a chance tell us if any of these cleaning methods will be obsolete at some point.